cmd/gradleExecuteBuild.go

```diff
@@ -67,14 +71,20 @@ initscript {
     }
   }
   dependencies {
-    classpath "com.cyclonedx:cyclonedx-gradle-plugin:1.5.0"
+    classpath "org.cyclonedx:cyclonedx-gradle-plugin:1.7.0"
   }
 }
 
 rootProject {
     apply plugin: 'java'
     apply plugin: 'maven'
     apply plugin: org.cyclonedx.gradle.CycloneDxPlugin
+
+    cyclonedxBom {
+	outputName = "` + gradleBomFilename + `"
+	outputFormat = "xml"
+	schemaVersion = "1.3"
```

Comment:
On Maven we use schema version 1.2, should we align this?

Reply:
I would suggest to use the latest available schema version for each tool. Ideally every tool supports v1.4 (https://github.com/CycloneDX/specification/releases) but I'm not sure if this is true

Reply:
should we change it with this PR or is it another feature/improvement?

Reply:
Would be great if it is ok for you

Reply:
https://www.npmjs.com/package/@cyclonedx/bom does not support schema version 1.4 (newest version is 1.3)